docs/installation-guide.md

```diff
@@ -92,77 +96,87 @@ To get started with CarbonData : [Quick Start](quick-start-guide.md), [DDL Opera
 
    The following steps are only for Driver Nodes. (Driver nodes are the one which starts the spark context.)
 
-* [Build the CarbonData](https://cwiki.apache.org/confluence/display/CARBONDATA/Building+CarbonData+And+IDE+Configuration) project and get the assembly jar from "./assembly/target/scala-2.10/carbondata_xxx.jar" and put in the ``"<SPARK_HOME>/carbonlib"`` folder.
+1. [Build the CarbonData](https://cwiki.apache.org/confluence/display/CARBONDATA/Building+CarbonData+And+IDE+Configuration) project and get the assembly jar from `./assembly/target/scala-2.1x/carbondata_xxx.jar` and copy to `<SPARK_HOME>/carbonlib` folder.
```

Comment:
1.please cite `https://github.com/apache/incubator-carbondata/blob/master/build/README.md` since wiki link has been invalid.
2.suggest change `./assembly/target/scala-2.1x/carbondata_xxx.jar` to `./assembly/target/scala-2.xx/carbondata_xxx.jar` as @chenliang613 mentioned above.